### DISCUSSION TITLE
Hotline after end-of-development

### DISCUSSION THREAD

**Discussion by MikeMcC399** [corona-warn-app/cwa-website#3517] [Q&A] [answered]
If the app hotline is shut down after the end of development on May 31, 2023, then it would need to be removed from the website.

![image](https://github.com/corona-warn-app/cwa-website/assets/66998419/aea5bb63-bbcf-45b7-9540-78602b3ec4f5)

Also [cypress/e2e/hotline.cy.j](https://github.com/corona-warn-app/cwa-website/blob/master/cypress/e2e/hotline.cy.js) should be disabled and deleted.

The hotline is not mentioned in the FAQ section https://www.coronawarn.app/en/faq/results/#ramp_down_effects. The repo https://github.com/corona-warn-app/cwa-hotline is however archived.

**Comment by Ein-Tim** [comment]
I will raise this topic internally, thanks Mike!

**Reply by MikeMcC399** [reply]
@Ein-Tim 

If you call the number 0800 7540001 which used to be the technical hotline for CWA you hear a message that the number is incomplete and you should call the directory enquiries (Auskunft).

So I have opened a new issue https://github.com/corona-warn-app/cwa-website/issues/3527 as this is no longer an open question or suggestion, it now requires a clean up on the web site.

**Reply by Ein-Tim** [reply]
@MikeMcC399 I already relayed your issue to the team and I expect that the reference in the footer will be removed today. I'm unsure whether it is necessary to add a FAQ entry informing the users that the hotline has shutdown to the [End of Development](https://coronawarn.app/en/faq/results/?search=&topic=end_of_development) section of the FAQs, what do you think?

**Reply by MikeMcC399** [reply]
> I'm unsure whether it is necessary to add a FAQ entry informing the users that the hotline has shutdown to the [End of Development](https://coronawarn.app/en/faq/results/?search=&topic=end_of_development) section of the FAQs, what do you think?

I think a short FAQ should be added:

1) The app is staying in the stores until mid June 2023, and there is now no hotline for it.
2) The new version of the app CWA Android 3.2.1 says "Please don't delete the app; you will be notified if it is reactivated or otherwise enhanced".

so it is not necessarily obvious that there is now no hotline despite the above.

**Reply by Ein-Tim** [reply]
@MikeMcC399 Okay, thanks for sharing your thoughts!

**Reply by MikeMcC399** [reply, marked as answer]
See PR https://github.com/corona-warn-app/cwa-website/pull/3529

**Comment by MikeMcC399** [comment]
@Ein-Tim 

See also comment in https://github.com/corona-warn-app/cwa-website/issues/3527#issuecomment-1572027136 regarding adding FAQ information.
